Products/ZenModel/migrate/Migrate.py

```diff
@@ -414,7 +414,7 @@ def orderedSteps(self):
     def list(self):
         print ' Ver      Name        Description'
         print '-----+---------------+-----------' + '-'*40
-        for s in self.allSteps:
+        for s in self.determineSteps():
```

Comment:
This looks suspect to me. 
If we are unit-testing a service migration, why do we need this change?